src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/ExternalUserController.kt

```diff
@@ -0,0 +1,44 @@
+package uk.gov.justice.digital.hmpps.manageusersapi.resource
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.manageusersapi.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.manageusersapi.service.UserGroupService
+
+@RestController
+@RequestMapping("/externalusers")
+class ExternalUserController(
+  private val userGroupService: UserGroupService
+) {
+
+  @GetMapping("/me/assignable-groups")
+  @Operation(
+    summary = "Get list of assignable groups.",
+    description = "Get list of groups that can be assigned by the current user."
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "200",
+        description = "OK"
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorized.",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class)
+          )
+        ]
+      )
+    ]
+  )
+  fun assignableGroups(): List<UserGroup> = userGroupService.getMyAssignableGroups()
+}
```

Comment:
Not important - but you don't need the return type

Reply:
Removed